test/simple/content/about-us.md

```diff
@@ -1,8 +1,7 @@
-{
-  "title": "About Us",
-  "published": "2015-06-06T09:49:08.789Z",
-  "type": "page"
-}
+---
+title: About Us
+published: 2015-06-06T09:49:08.789Z
+type: page
```

Comment:
YAML > JSON.
Or not. Whatever. But at least you don't need to worry about commas or quotes.